spec/support/shared_entity_specs.rb

```diff
@@ -70,3 +70,19 @@ def validate_values(value, parsed_value, type)
     end
   end
 end
+
+shared_examples "a relayable Entity" do
+  let(:instance) { described_class.new(data) }
+
+  context "signatures generation" do
+    it "computes correct signatures for the entity" do
+      xml = DiasporaFederation::Salmon::XmlPayload.pack(instance)
+      hash = DiasporaFederation::Salmon::XmlPayload.hash(xml.at_xpath("post/*[1]"))
```

Comment:
Well, the idea is to check how XmlPayoad.pack updates signature with overloaded by Relayable to_xml function. So I can wipe ```hash``` but I can't wipe ```pack```.